### PR TITLE
Added kind attribute in resources element in one of the example of Po…

### PR DESCRIPTION
**Pull request by viveksahu26**
…licy

Signed-off-by: viveksahu26 <vivekkumarsahu650@gmail.com>

### CODE REVIEW

**Review by vyankyGH — APPROVED** [review]

File: content/en/docs/Writing policies/match-exclude.md [hunk 1]
```diff
@@ -238,6 +238,8 @@ spec:
     - name: match-criticals-except-given-rbac
       match:
         resources:
+	  kinds:
+	  - Pod
```
Comment (RinkiyaKeDad) [inline content/en/docs/Writing policies/match-exclude.md:242]: What are these red colored blocks before the text?

Comment (viveksahu26) [reply content/en/docs/Writing policies/match-exclude.md:242]: Exactly, I also don't know. But, I think it's representing `tab` space.

Comment (RinkiyaKeDad) [reply content/en/docs/Writing policies/match-exclude.md:242]: Remove that please. I think just deleting it and adding it normally should fix it.

Comment (viveksahu26) [reply content/en/docs/Writing policies/match-exclude.md:242]: In the YAML file, indentation is necessary. And if I remove this indentation i.e. tab space, an error will occur in the YAML file.

Comment (chipzoller) [reply content/en/docs/Writing policies/match-exclude.md:242]: You need to check your IDE. A tab should be rendered as two spaces in YAML. This suggests it's rendered as a tab character which it shouldn't be.

Comment (viveksahu26) [reply content/en/docs/Writing policies/match-exclude.md:242]: Btw, I talked about that tab space from a general point of view. But, actually, I have used 2 spaces. 

Screenshot (587)

Comment (chipzoller) [reply content/en/docs/Writing policies/match-exclude.md:242]: That screenshot doesn't distinguish between those characters. VS Code, for example, will with the YAML addon.

Comment (viveksahu26) [reply content/en/docs/Writing policies/match-exclude.md:242]: Yes, @chipzoller when I open it in VS Code, I can easily visualize the exact problem. I don't why IDE to IDE these indentation differs. 

Screenshot (589)

File: content/en/docs/Writing policies/match-exclude.md [hunk 2]
```diff
@@ -238,6 +238,8 @@ spec:
     - name: match-criticals-except-given-rbac
       match:
         resources:
+	        kinds:
```
Comment (RinkiyaKeDad) [inline content/en/docs/Writing policies/match-exclude.md:241]: The red block is still there 😅 are you able to select that portion and remove it?

Comment (viveksahu26) [reply content/en/docs/Writing policies/match-exclude.md]: @RinkiyaKeDad , can you please check. I think I have removed the red portion.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**